gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -60,17 +60,31 @@ <h2 class="title">Affected packages</h2>
   <spicy-sections
       class="vulnerability-packages{% if vulnerability.affected|should_collapse %} force-collapse{% endif %}">
     {% for affected in vulnerability.affected %}
+      {% if affected['package']['ecosystem'] %}
       <h2 class="package-header">
         <span class="vuln-ecosystem spicy-sections-workaround">{{ affected['package']['ecosystem'] }}</span>
         <span class="vuln-title-divider spicy-sections-workaround">/</span>
         <span class="vuln-name spicy-sections-workaround">{{ affected['package']['name'] }}</span>
       </h2>
+      {% endif %}
       <div class="mdc-layout-grid">
-        <p class="subtitle">{{ affected['package']['name'] }}</p>
+        <div class="vulnerability-package-subsection mdc-layout-grid__inner">
+          <h3 class="mdc-layout-grid__cell--span-3">
+            Package Info
+          </h3>
+          <div class="mdc-layout-grid__cell--span-9">
+            <dl>
+              <dt>Package Name</dt>
+              <dd>{{ affected['package']['name'] or 'Unknown' }}</dd>
+              <dt>Repo URL</dt>
+              <dd><a href="{{ vulnerability.repo }}">{{ vulnerability.repo }}</a></dd>
```

Comment:
Thinking more on this, it's possible that there could be multiple repo URLs for the same tab/affected entry. We should handle such a case more gracefully, at least as a follow up item.

Reply:
Let's add a TODO somewhere on this.

Reply:
Done in [2ea079e](https://github.com/google/osv.dev/pull/1447/commits/2ea079ebed15318614b5ea12bdae826de01d4e14)